services/api/Tweek.ApiService/appsettings.json

```diff
@@ -26,10 +30,14 @@
         }
     },
     "UseAddon": {
-        "Context": "RedisContext",
+        "Context": "MultiContext",
         "Rules": "ManagementRules",
         "Metrics": "ApplicationInsights; GraphiteReporter"
     },
+    "MultiContext": {
+        "Readers": "Redis",
```

Comment:
The readers/writers can use the same addon, and we'll do mapping by assembly (assembly of driver <-> assembly of addon) instead of attribute